packages/browser/src/tracekit.ts

```diff
@@ -911,6 +912,21 @@ TraceKit._computeStackTrace = (function _computeStackTraceWrapper() {
     };
   }
 
+  function popFrames(stacktrace: any, popSize: number): any {
```

Comment:
Is there a reason not to use default value, then you could remove the number type check?!
```suggestion
  function popFrames(stacktrace: any, popSize: number = 0): any {
```

Reply:
It's Tracekit, so we have to be _very_ defensive in here. It's guarding us in case React breaks something, or someone decides to put `framesToPop` property on their error objects.